flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterTableAddConstraint.java

```diff
@@ -43,8 +43,11 @@
      * @param pos Parser position
      */
     public SqlAlterTableAddConstraint(
-            SqlIdentifier tableID, SqlTableConstraint constraint, SqlParserPos pos) {
-        super(pos, tableID);
+            boolean ifExists,
```

Comment:
Maybe change to `SqlAlterTableAddConstraint(SqlParserPos, SqlIdentifier, SqlTableConstraint, boolean)`?